js/piwik.js

```diff
@@ -6251,7 +6260,7 @@ if (typeof window.Matomo !== 'object') {
             this.enableHeartBeatTimer = function (heartBeatDelayInSeconds) {
                 heartBeatDelayInSeconds = Math.max(heartBeatDelayInSeconds || 15, 5);
                 configHeartBeatDelay = heartBeatDelayInSeconds * 1000;
-
+ console.log("here");
```

Comment:
haven't tested it but fyi there is a console.log call here

Reply:
@tsteur ah, sorry about that. let me remove this.